pkg/catalogsourceconfig/registry.go

```diff
@@ -96,7 +96,7 @@ func (r *registry) GetAddress() string {
 // ensureDeployment ensures that registry Deployment is present for serving
 // the the grpc interface for the packages from the given operatorSources
 func (r *registry) ensureDeployment(operatorSources string) error {
-	registryCommand := getCommand(r.csc.Spec.Packages, operatorSources)
+	registryCommand := getCommand(r.csc.GetPackages(), operatorSources)
```

Comment:
You should also fix `getCommand()` and have all that string fixup happen in just one place.

Reply:
Addressed.